sdk/react/src/markers/IconMarker/IconMarker.test.js

```diff
@@ -1,32 +1,67 @@
 // @flow
 
-import * as React from 'react';
-import renderer from 'react-test-renderer';
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native';
+import TestRenderer from 'react-test-renderer';
 
 import IconMarker from './IconMarker';
 
 describe('IconMarker', () => {
   it('renders matching snapshot', () => {
-    const tree = renderer.create(
+    const testRenderer = TestRenderer.create(
```

Comment:
`testRenderer` looks confusing as it's not a renderer but its instance.
As for me, `testInstance`, `element` or `wrapper` are more convenient

Reply:
Agree, I spent sometime on naming and just took what Facebook uses. Will use `wrapper` as common name in Enzyme.

Reply:
Fixed